### PR TITLE
Get the first account, default preferred

### PR DESCRIPTION
**Pull request by jSylvestre**

### CODE REVIEW

File: Anlab.Mvc/Services/DocumentSigningService.cs [hunk 1]
```diff
@@ -56,7 +56,7 @@ public async Task<string> SendForEmbeddedSigning(int orderId)
 
         var user = GetAccountInfo(authToken);
 
-        var account = user.Accounts[0]; // TODO: what if there are multiple accounts?  need to select one somehow perhaps
+        var account = user.Accounts.OrderByDescending(a => a.IsDefault).First(); // TODO: what if there are multiple accounts?  need to select one somehow perhaps
```
Comment (srkirkland) [inline Anlab.Mvc/Services/DocumentSigningService.cs:59]: Just change to `user.Accounts.Single(a => a.IsDefault)`

**Review by srkirkland — CHANGES_REQUESTED** [review]

File: Anlab.Mvc/Services/DocumentSigningService.cs (hunk 1 above)
Comment (jSylvestre) [reply Anlab.Mvc/Services/DocumentSigningService.cs:59]: Ok. I just wasn't sure every user had one that was default.
Then debugging, I saw it was your account :)

Comment (jSylvestre) [reply Anlab.Mvc/Services/DocumentSigningService.cs:59]: @srkirkland Ugg, it is a string.

Comment (srkirkland) [reply Anlab.Mvc/Services/DocumentSigningService.cs]: @jSylvestre oh yea, but don't use string.equals -- EF Core doesn't know how to use that so it doesn't translate like you think.  Just use `Single(a => a.IsDefault == "true");`.  SQL is case insensitive anyway.

Comment (srkirkland) [reply Anlab.Mvc/Services/DocumentSigningService.cs]: for more details if curious: https://learn.microsoft.com/en-us/ef/core/miscellaneous/collations-and-case-sensitivity#translation-of-built-in-net-string-operations

**Review by srkirkland — APPROVED** [review]

**Pull request merged**